integration/ipam/ipam.go

```diff
@@ -37,7 +39,20 @@ func GenerateFlannelNetwork(ctx context.Context, clusterID string, crdStorage mi
 
 	cidrMask := net.CIDRMask(flannelCidrSize, 32)
 
```

Comment:
Maybe that's helpful: https://github.com/giantswarm/microstorage/tree/master/retrystorage

Reply:
Oh, that would be cleaner approach. I'll refactor this in the morning.